### DISCUSSION TITLE
Unable to download from Niconico

### DISCUSSION THREAD

**Discussion by Browsinround** [streamlink/streamlink-twitch-gui#987] [Q&A]
Was trying to download a time-shifted stream that my account has access to but was unable to do so.

I used the following command

`streamlink --niconico-email myemail  --niconico-password mypass --output "Filename.mkv" link best
`

so far I had two outcomes. 

1. I recieved an unable to refresh playlist error (even though there was no playlist)
2. Download completes but the screen is black beyond the free portion

**Comment by bastimeyer** [comment]
Wrong repo.
